filesystems/filesystems.tex

```diff
@@ -766,19 +785,18 @@ \section{Virtual filesystems and other filesystems}
 POSIX systems, such as Linux and Mac OS X (which is based on BSD) include several virtual filesystems that are mounted (available) as part of the file-system.
 Files inside these virtual filesystems do not exist on the disk.
 They are generated dynamically by the kernel when a process requests a directory listing.
-Linux provides 3 main virtual filesystems.
+Linux provides 3 main virtual filesystems:
 
 \begin{verbatim}
-/dev  - A list of physical and virtual devices (for example network card, cdrom, random number generator)
-/proc - A list of resources used by each process and (by tradition) set of system information
-/sys - An organized list of internal kernel entities
+    /dev  - A list of physical and virtual devices (for example network card, cdrom, random number generator)
+    /proc - A list of resources used by each process and (by tradition) set of system information
+    /sys - An organized list of internal kernel entities
 \end{verbatim}
```

Comment:
This should probably be a list or table.

Reply:
I thought so too, but I wasn't sure.